src/ComponentFolders/SignUpPage/SignUp.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import Image from './Images/SofaImage.svg';
+import lineImage from './Images/line.svg';
+import FacebookLogo from './Images/FacebookLogo.svg';
+import GoogleLogo from './Images/GoogleLogo.svg';
+
+function SignUp() {
+  const [signUpData, setSignUpData] = useState({
+    userFirstName: '',
+    userLastName: '',
+    userEmail: '',
+    userConfirmEmail: '',
+    userPassword: '',
+    userCondirmPassword: '',
+    dayOfBirth: '',
+    monthOfBirth: '',
+    yearOfYear: '',
+  });
+
+  function handleOnClick(e) {
+    const { value, name } = e.target;
+    return setSignUpData((prevObj) => {
+      return {
+        ...prevObj,
+        [name]: value,
+      };
+    });
+  }
+
+  return (
+    <div className="h-screen flex justify-center content-center md:flex-wrap max-[767px]:flex-wrap gap-x-20">
```

Comment:
![Screenshot_20221228_191717](https://user-images.githubusercontent.com/110013797/209843165-27c5086f-77d7-447a-970e-ad93609ca4aa.png)
half of the image disappears when I change the size of the page :D something wrong with the responsiveness </3

Reply:
![Screenshot 2022-12-29 150530](https://user-images.githubusercontent.com/86560837/209949118-43a5c964-3385-4173-9161-94c0b245238c.jpg)

Reply:
On my side it's fine, could it be a browser problem ?